clj-refactor.el

```diff
@@ -2184,9 +2184,10 @@ FEATURE is either :clj or :cljs."
 (defun cljr--magic-requires-lookup-alias ()
   "Return (alias (ns.candidate1 ns.candidate1)) if we recognize
 the alias in the project."
-  (let ((short (buffer-substring-no-properties
-                (cljr--point-after 'paredit-backward)
-                (1- (point)))))
+  (let ((short (->> (buffer-substring-no-properties
```

Comment:
nope, does not work here as `short` needs to be stripped of the `/`. using `(cider-symbol-at-point)` and then stripping the `/` manually does not feel right.